gateway/tests/fixtures/fixtures.json

```diff
@@ -125,12 +128,15 @@
       "env_vars": "{\"PROGRAM_ENV1\": \"VALUE1\", \"PROGRAM_ENV2\": \"VALUE2\"}",
       "instances": [
         100
+      ],
+      "trial_instances": [
+        100
       ]
     }
   },
   {
     "model": "api.job",
-    "pk": "1a7947f9-6ae8-4e3d-ac1e-e7d608deec82",
+    "pk": "57fc2e4d-267f-40c6-91a3-38153272e764",
```

Comment:
These changes @paaragon about the `pk` in the fixtures was because the UUID between Functions and Jobs were being reused and it was created me some problems so I just updated them.